src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.model;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
+
+    Operation(String code) {
+        this.code = code;
+    }
+
+    public static Operation fromCode(String code) {
+        switch (code) {
```

Comment:
switch is a bad practice. As well as hardcoding cases.
Better use loop over `values()`